src/components/Project/index.js

```diff
@@ -154,51 +154,59 @@ class Project extends React.Component {
       return ["waiting", "running"].includes(release.state)
     }).length
 
+    let currentEnv = this.props.store.app.currentEnvironment.key
     this.props.store.app.leftNavItems = [
         {
           key: "10",
           icon: <FeaturesIcon />,
           name: "Features",
-          slug: this.props.match.url + "/features",
+          slug: `${props.match.url}/${currentEnv}/features`,
           count: deployableFeatures,
         },
         {
           key: "20",
           icon: <ReleasesIcon />,
           name: "Releases",
-          slug: this.props.match.url + "/releases",
+          slug: `${props.match.url}/${currentEnv}/releases`,
           count: releasesQueued,
           badgeColor: "secondary",
         },
         {
           key: "30",
           icon: <ServicesIcon />,
           name: "Services",
-          slug: this.props.match.url + "/services",
+          slug: `${props.match.url}/${currentEnv}/services`,
         },
         {
           key: "40",
           icon: <SecretIcon />,
           name: "Secrets",
-          slug: this.props.match.url + "/secrets",
+          slug: `${props.match.url}/${currentEnv}/secrets`,
         },
         {
           key: "50",
           icon: <ExtensionsIcon />,
           name: "Extensions",
-          slug: this.props.match.url + "/extensions",
+          slug: `${props.match.url}/${currentEnv}/extensions`,
         },
         {
           key: "60",
           icon: <SettingsIcon />,
           name: "Settings",
-          slug: this.props.match.url + "/settings",
+          slug: `${props.match.url}/${currentEnv}/settings`,
         },
     ];
   }
 
   componentWillReceiveProps(nextProps) {
-    this.setLeftNavProjectItems(nextProps)
+    let currentEnv = this.props.store.app.currentEnvironment
+    if (!!currentEnv && !!currentEnv.key) {
```

Comment:
I will make an issue to change all other truthiness checks in this code to this format